funcbench/main.go

```diff
@@ -307,10 +306,11 @@ func getTargetInfo(repo *git.Repository, target string) plumbing.Hash {
 
 type commander struct {
 	verbose bool
+	ctx     context.Context
```

Comment:
Is it okay to have the context in the struct for `commander`? what if we want to cancel the context from something that's using it. I am a little unsure about this. :(